### PR TITLE
Update fftw3 0.7.1 to OPAM 1.2 

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

**Comment by dsheets** [comment]
This contains `oasis2opam`, too.

File: packages/fftw3/fftw3.0.7.1/opam [hunk 1]
```diff
@@ -1,10 +1,12 @@
-opam-version: "1"
+opam-version: "1.2"
 name: "fftw3"
 version: "0.7.1"
```
Comment (dsheets) [inline packages/fftw3/fftw3.0.7.1/opam]: Could you please remove `name` and `version`?

**Comment by Chris00** [comment]
Gosh—doing too many things concurrently...  Fixed.

**Comment by Chris00** [comment]
Transient failure.

**Comment by samoht** [comment]
I've restarted the failed job

**Comment by Chris00** [comment]
Download problem, again!

File: packages/fftw3/fftw3.0.7.1/opam [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+opam-version: "1.2"
 authors: ["Christophe Troestler <Christophe.Troestler@umons.ac.be>"]
 maintainer: "Christophe.Troestler@umons.ac.be"
 license: "LGPL-2.1 with OCaml linking exception"
 homepage: "https://github.com/Chris00/fftw-ocaml"
+dev-repo: "https://github.com/Chris00/fftw-ocaml.git"
+bug-reports: "https://github.com/Chris00/fftw-ocaml/issues"
 build: [
   ["./configure" "--prefix" prefix "--sbindir=%{lib}%/fftw3/sbin" "--libexecdir=%{lib}%/fftw3/libexec" "--sysconfdir=%{lib}%/fftw3/etc" "--sharedstatedir=%{lib}%/fftw3/com" "--localstatedir=%{lib}%/fftw3/var" "--libdir=%{lib}%/fftw3/lib" "--includedir=%{lib}%/fftw3/include" "--datarootdir=%{lib}%/fftw3/share"]
   [make]
```
Comment (dsheets) [inline packages/fftw3/fftw3.0.7.1/opam]: `[make "install"]` could go into the `install` section but I've merged this anyway.

Comment (Chris00) [inline packages/fftw3/fftw3.0.7.1/opam]: On Mon, 6 Jul 2015 05:26:57 -0700, David Sheets wrote:
> 
> [make "install"] could go into the install section but I've merged this anyway.

Right.  Will be for the next version.

**Pull request merged**